_posts/2021-01-03-flow-matching.md

```diff
@@ -18,12 +18,40 @@ published: true
 - [TUM Tutorial - General Matching (Edmonds' Blossom Algorithm](https://www-m9.ma.tum.de/graph-algorithms/matchings-blossom-algorithm/index_en.html){:target="\_blank"} 
 - [Quora Answer - Complexity of Edmond Karp](https://www.quora.com/Is-there-a-simple-intuitive-way-to-explain-why-the-Edmonds-Karp-max-flow-algorithm-takes-O-EV-2/answer/Tadeusz-Panda){:target="\_blank"} 
 - [Competitive Programmer's Handbook - Flows and Cuts](https://raw.githubusercontent.com/baps-bgd/baps-bgd.github.io/master/_files/Competitive-Programmer%E2%80%99s-Handbook.pdf#page=191){:target="\_blank"} 
+- [Min Cost Any Circular Flow Code](https://github.com/stjepang/snippets/blob/master/circulation.cpp){:target="\_blank"} 
+
 
 [](){:target="\_blank"} 
 [](){:target="\_blank"} 
 [](){:target="\_blank"} 
 
 
-## Related Problems
+## Max Flow Problems
```

Comment:
Not sure about this types of subtopic tags. I will prefer to keep everything under one roof. A little bit of mistery is needed for learning in my opinion.